docs/docs/administration/email-notification.md

```diff
@@ -0,0 +1,67 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Email Notifications
+
+Immich supports the option to send notifications via email, this option is supported for:
+
+- Creating a new user .
+- A user is added to a shared album (send email to the newly shared user).
+- A shared album is added with new assets (send an email to all shared users beside the one that is adding assets).
+
+The easiest way to set up a mail server for notifications is through Google, you may want to avoid using Gmail for privacy reasons or as part of your degoogle plan, you can use any SMTP server that can support login using name and password.
+
+## Create an app password
+
+First, it is recommended to create a dedicated Google account for your Immich instance, and not to mix it with your main Google account.
```

Comment:
Is this your opinion? We should be subjective in writing documentation

Reply:
We should *not* be subjective :D

Reply:
I put it in a :::tip:::, if you think it should be removed, tell me.